src/main/scala/uk/gov/nationalarchives/tdr/api/model/file/FileType.scala

```diff
@@ -0,0 +1,38 @@
+package uk.gov.nationalarchives.tdr.api.model.file
+
+import uk.gov.nationalarchives.tdr.api.graphql.DataExceptions.InputDataException
+
+object FileType {
```

Comment:
This is probably too late since we have a column with this name, but maybe the name could be something like "ItemType", "ObjectType", "ResourceType" or "ContentType"? "FileType" sounds like the type of file e.g. pdf, png, mp3 etc.

Reply:
I've changed the object name to NodeType as per the slack discussion.